Formula/afuse.rb

```diff
@@ -29,6 +29,18 @@ def install
     system "make", "install"
   end
 
+  def caveats
+    on_macos do
+      <<~EOS
+        Reasons for disabling this formula can be found here:
+          https://github.com/Homebrew/homebrew-core/pull/64491
+
+        For a replacement tap, search for "FUSE" in:
+          https://docs.brew.sh/Interesting-Taps-and-Forks
```

Comment:
```suggestion
       An external tap may provide a replacement formula. Here are a few interesting ones:
          https://docs.brew.sh/Interesting-Taps-and-Forks
```

Reply:
Or just:
```
    An external tap may provide a replacement formula. See:
      https://docs.brew.sh/Interesting-Taps-and-Forks
```

Reply:
Would love this to always say something about "note: these are not supported, maintained or endorsed by Homebrew"